### DISCUSSION TITLE
Border-color  on active, focus and hover

### DISCUSSION THREAD

**Discussion by javierfuentesm** [chakra-ui/chakra-ui#3162] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19244527/105799446-2c4ef700-5f5a-11eb-98b4-cd7fef43dcbb.png)

is there any other way ?? to achieve this effect? I have been trying to change the border-color when hover, focus and on active , but i couldnt figure it out how to do it.

This is the css 
![image](https://user-images.githubusercontent.com/19244527/105799555-72a45600-5f5a-11eb-847b-424265fda44c.png)

**Comment by javierfuentesm** [comment, marked as answer]
Nevermind I solved it

**Reply by with-heart** [reply]
Please explain how you solved it for anyone coming after!

**Comment by erikP3K** [comment]
For the people who want a quick fix. ;D
Add this to e.g. a chakra-ui Box component:

`_hover={{color: '#0070f3', borderColor: '#0070f3'}}`
